### ISSUE TITLE
(4.9.0) strange moves and floating layers when using Print Sequence: One at a Time

### ISSUE THREAD

**Issue by PHL003** [Ultimaker/Cura#9713]
### Application Version

4.9.0

### Platform

Windows 10

### Printer

Biqu B1

### Reproduction steps

1. Placed object (stl file) containing 6 pins into Cura
2. Selected "Print Sequence": "One at a Time"
3. Repositioned objects to avoid overlapping print head areas
4. Changed "Build Plate Adhesion Type" from Brim to Raft
5. Clicked "Slice"
(note: this issue only appears when selecting "One at a Time" and "Raft" in combination; All at Once or Brim not affected)

### Actual results

1. Preview when moved to top layer shows "floating" layers above 3 of the 6 pins
![6 pins_raft](https://user-images.githubusercontent.com/83244562/116286504-9791aa80-a78f-11eb-8e28-08d77dedfbbc.jpg)
even deleting 1 of the pins and replicating one of the unaffected pins does not resolve the issue (the cloned pin is shown as normal but the one it was cloned from now has the floating top layer)
2. when going through the preview layer by layer there are unexpected moves at the changeover for each pin, for example pin 1 is completed at layer 52, raft 2 starts at layer 53, but at layer 56 all of the pins are shown as completed with the floating layer above pin 2, and at layer 58 the preview returns to pin 1 before going back to pin 2 at layer 59
![6 pins_raft_layer56](https://user-images.githubusercontent.com/83244562/116286640-bc861d80-a78f-11eb-8c2f-623dfd6608a7.jpg)
![Uploading 6 pins_raft_layer58.jpg…]()
(I have more screenshots available if you need)

### Expected results

no floating (whether actual or just in preview), each item in a "One at a Time" selection should be completed before moving on to the next item, and never returned to again

### Checklist of files to include

- [ ] Log file
- [X] Project file

### Additional information & file uploads

[BB1_Hydraulic_pin_large_x6.zip](https://github.com/Ultimaker/Cura/files/6386504/BB1_Hydraulic_pin_large_x6.zip)
Project file attached

**Comment by fvrmr** [comment]
Hi @PHL003 thank you for your bug report. 
This is a duplicate of https://github.com/Ultimaker/Cura/issues/7412 
And we have a ticket on our backlog for this. 

I will close this ticket since it is a duplicate.
